### PR TITLE
Fuzzer cleanup

### PR DESCRIPTION
**Pull request by cujomalainey**
* Delete old fuzzer code
* Fix overflow in posix fuzzers

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@ranj063 pls review.

File: src/platform/posix/ipc.c [hunk 1]
```diff
@@ -30,7 +30,7 @@ extern uint8_t *posix_fuzz_buf, posix_fuzz_sz;
 // thinking/experimentation about how much fuzzing we want to do at a
 // time...
 static uint8_t fuzz_in[65536];
-static uint8_t fuzz_in_sz;
+static size_t fuzz_in_sz;
```
Comment (andyross) [inline src/platform/posix/ipc.c:33]: Nitpick: this bug is actually pretty subtle and maybe deserves more explanation in the commit message, if for no better reason than to sell how clever a catch it was.

**Review by andyross — APPROVED** [review]
Whine about verbosity, but looks good here

File: src/platform/posix/ipc.c (hunk 1 above)
Comment (cujomalainey) [reply src/platform/posix/ipc.c:33]: lol fair

**Pull request merged**